rotkehlchen/tests/api/test_balances.py

```diff
@@ -973,3 +983,108 @@ def query_blockchain_balance(num: int) -> Any:
             A_USDT.identifier: {'amount': '3', 'usd_value': '54'},
         }
         assert one_time_query_result == query_blockchain_balance(4)
+
+
+@pytest.mark.parametrize('number_of_eth_accounts', [2])
+@pytest.mark.parametrize('btc_accounts', [[UNIT_BTC_ADDRESS1, UNIT_BTC_ADDRESS2]])
+@pytest.mark.parametrize('added_exchanges', [(Location.BINANCE, Location.POLONIEX)])
+def test_query_balances_with_threshold(
+        rotkehlchen_api_server_with_exchanges: 'APIServer',
+        ethereum_accounts: list['ChecksumEvmAddress'],
+        btc_accounts: list['BTCAddress'],
+) -> None:
+    """Test that balance filtering by USD value threshold works for all balance types"""
+    rotki = rotkehlchen_api_server_with_exchanges.rest_api.rotkehlchen
+
+    manually_tracked_balances = [
+        ManuallyTrackedBalance(
+            identifier=-1,
+            asset=A_EUR,
+            label='Small EUR',
+            amount=FVal('1'),
```

Comment:
use `ONE` (we have this constant) instead of calling FVal

```suggestion
            amount=ONE,
```